src/pc/djui/djui_inputbox.c

```
@@ -636,14 +648,15 @@ static void djui_inputbox_destroy(struct DjuiBase* base) {
 struct DjuiInputbox* djui_inputbox_create(struct DjuiBase* parent, u16 bufferSize) {
     struct DjuiInputbox* inputbox = calloc(1, sizeof(struct DjuiInputbox));
     struct DjuiBase* base         = &inputbox->base;
+    struct DjuiTheme* theme       = gDjuiThemes[configDjuiTheme];
     inputbox->bufferSize = bufferSize;
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)